### DISCUSSION TITLE
Rune pouch & spellbook check on toa entrance, similar to TOB

### DISCUSSION THREAD

**Discussion by Sebas613** [runelite/runelite#17889] [Ideas]
Hi,

It'd be great if we could get a check on the ToA entrance door that checks your current spellbook and whether the runes you're bringing match up. For example, if you're on ancients but dont have runes for blood spells, or are on arceus and don't have runes for thralls.

It can be frustrating enough in solo's when you realize at the first boss that you're lacking the correct spells, but for groups it'd be particularly painful.

**Comment by Felanbird** [comment]
Did you try [ToB Gear Checker](https://runelite.net/plugin-hub/show/tob-gear-checker) it claims to cover `spellbook, runes, and charges.`

Also maybe the [Toa Spellbook Reminder](https://runelite.net/plugin-hub/show/toa-spellbook-reminder) dev might be willing to expand the plugin to work in ToB

**Reply by YvesW** [reply]
FYI ToB Gear checker is quite outdated (and useless for that reason). I think OP is talking about the ToB QoL plugin by Damen that warns him if he's on the wrong spellbook/runes.

**Reply by Sebas613** [reply]
> Did you try [ToB Gear Checker](https://runelite.net/plugin-hub/show/tob-gear-checker) it claims to cover `spellbook, runes, and charges.`
> 
> Also maybe the [Toa Spellbook Reminder](https://runelite.net/plugin-hub/show/toa-spellbook-reminder) dev might be willing to expand the plugin to work in ToB

I'd like that ToB one but for ToA yes. I'm hunting fang at the moment, and the plugin would be great to avoid having to reset.
